happening/patterns/404.php

```diff
@@ -6,36 +6,36 @@
  * Inserter: no
  */
 ?>
-<!-- wp:cover {"url":"<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/assets/images/image-from-rawpixel-id-3306149_distorted.jpg","dimRatio":20,"isUserOverlayColor":true,"minHeight":100,"minHeightUnit":"vh","contentPosition":"top center","style":{"elements":{"link":{"color":{"text":"var:preset|color|base"}},"heading":{"color":{"text":"var:preset|color|base"}}},"spacing":{"padding":{"top":"3rem","bottom":"3rem"}}},"textColor":"base","layout":{"type":"constrained"}} -->
-<div class="wp-block-cover has-custom-content-position is-position-top-center has-base-color has-text-color has-link-color" style="padding-top:3rem;padding-bottom:3rem;min-height:100vh"><span aria-hidden="true" class="wp-block-cover__background has-background-dim-20 has-background-dim"></span><img class="wp-block-cover__image-background" alt="<?php esc_attr_e( 'Distorted image of a man running from behind', 'happening' ); ?>" src="<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/assets/images/image-from-rawpixel-id-3306149_distorted.jpg" data-object-fit="cover"/><div class="wp-block-cover__inner-container"><!-- wp:group {"align":"wide","style":{"dimensions":{"minHeight":"85vh"},"spacing":{"padding":{"top":"0rem","right":"0rem","bottom":"0rem","left":"0rem"},"margin":{"top":"0rem","bottom":"0rem"}}},"layout":{"type":"flex","orientation":"vertical","justifyContent":"stretch","verticalAlignment":"space-between","flexWrap":"wrap"}} -->
```

Comment:
I know this is CBT's fault but I think the Theme check plugin will fail because of this change, we want `esc_attr_e` instead of `esc_html_e` for alt attributes

Reply:
Yes, I remember that. I'll fix it.